js/app.js

```diff
@@ -133,6 +129,41 @@ function renderChart() {
     });
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////////////////////////////////
+
+// for lab 13:
+// Steps for local storage:
+// 	1. Setting items
+// 		a. JSON.stringify()  - turns javascript object into JSON object
+// 		b. localStorage.setItem("key", "value")
+// 	2. Getting items
+// 		a. localStorage.getItem("key")
+//      b. JSON.parse(variable)
+
+// function setAllProducts()
+function setItems(key, value){
+    // step 1a
+    let stringifyVersion = JSON.stringify(value);
+    // step 1b
+    localStorage.setItem(key, stringifyVersion);
+}
```

Comment:
I like that this is a reusable function, instead of just grabbing the value you need.